docs/_data/sidebar-fx.yml

```diff
@@ -40,27 +36,10 @@ options:
       - title: Semaphore
```

Comment:
I cannot add multiline suggestions for code that wasn't modified it seems.

`ConcurrrentVar`, `Promise` & `Semaphore` are deprecated in favor of KotlinX versions of these data types. KotlinX has the following types with the same semantics: `Channel()`, `CompletableDeferred()` and `Semaphore(..)`.

Reply:
Thanks @nomisRev !!